.sauce/config.yml

```diff
@@ -10,12 +10,13 @@ sauce:
       - e2e
       - release team
       - other tag
-    build: $CI_COMMIT_SHORT_SHA
+    build: "config-$CURRENT_TIME"
```

Comment:
`$CURRENT_TIME` does not exist in our environment. So this won't really do for our pipelines here.

Btw. not sure if you knew, but if you leave out `build` entirely, saucectl will automatically generate a build with a timestamp. So the above is entirely unnecessary.

Reply:
@alexplischke had no idea! very cool. Let me try

Reply:
@alexplischke excellent functionality! Fixed